### ISSUE TITLE
SQL worker can not start directly

### ISSUE THREAD

**Issue by zymap** [apache/pulsar#7390]
**Describe the bug**

After building master code, SQL worker can not start directly

**To Reproduce**
1. Build the master code
2. Execute `bin/pulsar sql-worker run`

**Expected behavior**

**Screenshots**
```
ERROR: Future versions of Presto will require Java 11 after March 2020.

You may temporarily continue running on Java 8 by adding the following
JVM config option:

    -Dpresto-temporarily-allow-java8=true
```

**Comment by jiazhai** [comment]
@zymap What is the version of pulsar you are using?

**Comment by enriched** [comment]
This is because the presto version got moved forward to `io.prestosql:332`. I think that adding `-Dpresto-temporarily-allow-java8=true` to the end of https://github.com/apache/pulsar/blob/master/conf/presto/jvm.config would do the trick.

**Comment by zymap** [comment]
Already fixed by #7898. close this issue.
